LinkedListCycleII.java

```diff
@@ -0,0 +1,63 @@
+import java.util.HashSet;
+/*
+
+Time complexity: O(n)
+Space complexity: O(n)
+Run on leetcode: yes
```

Comment:
Please specify what `n` indicates.

Reply:
Here, N specify the number of node in the Linked List.